DEVELOPER.md

```diff
@@ -404,24 +412,24 @@ To avoid long compilation times, give the VM plenty of RAM (8GB recommended) and
 You will need to setup port forwarding on VirtualBox to be able to ssh into the box which can be done as follows:
 
 1. Select the virtual machine you want to use and click "Settings"
-1. Click the "Network" tab
-1. Click the "Advanced" dropdown
-1. Click "Port Forwarding"
-1. Add a new rule in the popup. The only thing you will need is "Host Port" to be 22222 and "Guest Port" to be 22. Everything else can be left default (see screenshot below)
-1. Click "Ok"
+2. Click the "Network" tab
+3. Click the "Advanced" dropdown
+4. Click "Port Forwarding"
+5. Add a new rule in the popup. The only thing you will need is "Host Port" to be 22222 and "Guest Port" to be 22. Everything else can be left default (see screenshot below)
```

Comment:
```suggestion
5. Add a new rule in the popup. The only thing you will need is set "Host Port" to be 22222 and "Guest Port" to be 22. Everything else can be left default (see screenshot below)
```

Reply:
You can use all `1.`s in Markdown and let the generator automatically maintain the sequence, it is a common practice and makes adding additional items easier.

Reply:
reverted